frontend/src/lib/translations.ts

```diff
@@ -248,6 +248,7 @@ export const translationIds = {
 			"Sie sind bereits angemeldet. Melden sie sich zuerst ab um den Account zu wechseln.",
 		alertMessageTitle: "Fehler",
 		badCredentials: "Ungültige E-Mail-Adresse oder ungültiges Passwort",
```

Comment:
Could we use the existing `badCredentials` text here, or does `badData` have a different meaning? (just to avoid duplication if possible)